net/README.md

```diff
@@ -28,8 +28,9 @@ for it to work.
 
 ### Step2: Provide executable privileges and run installer script
 - `chmod +x net_demo_installer`
-- Run net_demo_installer script:
-`./net_demo_installer <server_1_ip_or_dns> <server_2_ip_or_dns> . . .`
+- Run net_demo_installer script. NOTE: Provide the network interface information on which the IP is configured (as mentioned in Step 6 of pre-requisites):
+
+         `./net_demo_installer <server1_ip_or_dns>:<server1_nw_if> <server2_ip_or_dns>:<server2_nw_if> . . .`
```

Comment:
Is it worth keeping this interface as eth1 when not specified; I am okay even without it, just wondering..